pyproject.toml

```diff
@@ -24,9 +24,9 @@ classifiers = [
 keywords = ["aiida", "firecrest"]
 requires-python = ">=3.9"
 dependencies = [
-    "aiida-core@git+https://github.com/chrisjsewell/aiida_core.git@aiida-firecrest#egg=aiida-core",
+    "aiida-core@git+https://github.com/khsrali/aiida-core.git@aiida-firecrest#egg=aiida-core",
```

Comment:
Is there PR in aiida-core that merges this? Would be useful for review but dont worry if it does not exist, I can also look at your commits

Reply:
Yes! there is a blocking [#PR6043](https://github.com/aiidateam/aiida-core/pull/6043) 
feel free to review that as well!
@mbercx was interested to check if that PR is working with hyper-Q. 
He was interested review after release of `aiida-core v2.6.0`

Reply:
note in [my fork](https://github.com/khsrali/aiida-core/tree/aiida-firecrest), that has already been merged with *latest* verion of `aiida-core` , that's why I replaced the hook from Chris's fork.

Reply:
it is now merged
```suggestion
    "aiida-core@git+https://github.com/aiidateam/aiida-core.git@954cbdd3",
```

Reply:
@agoscinski How to change it to the latest version of `aiida-core`? maybe that is better than the specific commit

Reply:
thanks!